fastapi_asyncapi/schema.py

```diff
@@ -0,0 +1,260 @@
+from typing import Any, Dict, List, Optional, Union
+
+from pydantic import AnyHttpUrl, BaseModel, EmailStr, Field
+from typing_extensions import Literal
+
+ASYNCAPI_VERSION = "2.3.0"
+
+
+class Contact(BaseModel):
+    name: Optional[str]
+    url: Optional[AnyHttpUrl]
+    email: Optional[EmailStr]
+
+
+class ExternalDocumentation(BaseModel):
+    description: Optional[str]
+    url: AnyHttpUrl
+
+
+class Tag(BaseModel):
+    name: str
+    description: Optional[str]
+    externalDocs: Optional[ExternalDocumentation]
+
+
+class License(BaseModel):
+    name: str
+    url: Optional[str]
+
+
+class Info(BaseModel):
+    title: str
+    version: str
+    description: Optional[str]
+    termsOfService: Optional[AnyHttpUrl]
+    contact: Optional[Contact]
+    license: Optional[License]
+
+
+Protocol = Literal[
+    "amqp",
+    "amqps",
+    "http",
+    "https",
+    "jms",
+    "kafka",
+    "kafka-secure",
+    "mqtt",
+    "secure-mqtt",
+    "stomp",
+    "stomps",
+    "ws",
+    "wss",
+]
+
+
+class ServerVariable(BaseModel):
+    enum: Optional[List[str]]
+    default: str
```

Comment:
https://www.asyncapi.com/docs/specifications/v2.3.0#serverVariableObject

```suggestion
    default: Optional[str]
```